src/bot/index.ts

```diff
@@ -96,12 +99,46 @@ export default class Bot extends Client {
   async onMessageCreate(message: Message | PartialMessage): Promise<void> {
     if (!message.inGuild()) return;
     this.stickys.run(message);
+  } // dis moi je chauffe ou je refroidis
+
+  async onGuildMemberUpdate(oldMember: GuildMember, newMember: GuildMember) {
+    if (oldMember.displayAvatarURL() !== newMember.displayAvatarURL()) {
+      if (!newMember.roles.cache.has(godfatherRoleId)) return;
+      const guild = this.guilds.cache.get(emojisGuildId);
```

Comment:
Pour la vérification utilise plutôt cela :)
```suggestion
  if (newMember.guild.id !== guildId) return;
  
  if (!newMember.roles.cache.has(godfatherRoleId)) return;
```